scrabble.js

```diff
@@ -1,15 +1,102 @@
+'use strict';
+
+const MAX = 7;
+
+const POINTS_VALUES = new Map([
+  ['A', 1], ['B', 3], ['C', 3], ['D', 2], ['E', 1], ['F', 4], ['G', 2], ['H', 4], ['I', 1],
+  ['J', 8], ['K', 5], ['L', 1], ['M', 3], ['N', 1], ['O', 1], ['P', 3], ['Q', 10], ['R', 1],
+  ['S', 1], ['T', 1], ['U', 1], ['V', 4], ['W', 4], ['X', 8], ['Y', 4], ['Z', 10]]);
 
 const Scrabble = {
-  score(word) {
 
+  getPointValue(letter) {
+    if (POINTS_VALUES.get(letter) === undefined) { throw new Error(`Invalid letter: ${letter}`); }
+    return POINTS_VALUES.get(letter);
+  },
+
+  isValidString(str) {
```

Comment:
I like this naming convention of `isValidString`... just by naming something starting with "is" helps me understand the point of the function, which is to check validity...
Maybe it should return `true` if it is valid?